libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/SalesforceAnalyticsManager.java

```diff
@@ -200,6 +204,13 @@ public static synchronized void setPublishFrequencyInHours(int publishFrequencyI
         }
     }
 
+    public static void setEventPublishBatchSize(int batchSize) {
+        if (batchSize <= 0) {
```

Comment:
I'd prefer that we rework this to not throw an exception. Instead, if input is `<=0`, simply return right away and continue to use the default of `100`. In `Javadoc` for this method, you can mention that valid input needs to be `> 0` if you want.